src/contrib/serializers/Akka.Serialization.Hyperion/HyperionSerializer.cs

```diff
@@ -70,7 +70,8 @@ public HyperionSerializer(ExtendedActorSystem system, HyperionSerializerSettings
                     preserveObjectReferences: settings.PreserveObjectReferences,
                     versionTolerance: settings.VersionTolerance,
                     surrogates: new[] { akkaSurrogate },
-                    knownTypes: provider.GetKnownTypes()));
+                    knownTypes: provider.GetKnownTypes(), 
+                    ignoreISerializable:true));
```

Comment:
I agree, this PR is not about using ignoreISerializable or not. Its about fixing the package. Default value is false, so set it at false. And lets have the discussion on when to turn this option on, a different time.